woocommerce-services.php

```diff
@@ -694,28 +693,35 @@ public function add_tracking_info_to_emails( $order, $sent_to_admin, $plain_text
 
 			// Generate a table row for each label
 			foreach ( $labels as $label ) {
-				$carrier = esc_html( $label['carrier_id'] );
-				$tracking = esc_html( $label['tracking'] );
+				$carrier = strtoupper( $label['carrier_id'] );
```

Comment:
Moving the `strtoupper()` call here broke the `switch()` case below. Let's either change the `case`s to be uppercase strings or call `strtoupper()` as late as possible.